frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -1131,8 +1138,9 @@ function ReaderPaging:onGotoPage(number, pos)
         else
             self.view:PanningUpdate(rect_s.x - self.view.visible_area.x, rect_s.y - self.view.visible_area.y)
         end
-    elseif number == self.current_page then
```

Comment:
I'm mildly concerned whether this using the post-`_goToPage` `self.current_page` value was by-design for some "readerpaging is a gnarly mess" corner-case, or a simple oversight ;).

Reply:
(I'm assuming `_goTopagze`, can, in fact, affect the value of `self.Current_page` here. I haven't actually double-checked that).

Reply:
Not familiar with the codepath, but I would tend to agree with your reading, so let's roll with it ;).